tools/unitctl/Cargo.lock

```diff
@@ -609,14 +715,48 @@ dependencies = [
  "itoa",
 ]
 
+[[package]]
+name = "http"
+version = "1.1.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "21b9ddb458710bc376481b842f5da65cdf31522de232c1ca8146abce2a358258"
+dependencies = [
+ "bytes",
+ "fnv",
+ "itoa",
+]
+
 [[package]]
 name = "http-body"
 version = "0.4.5"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "d5f38f16d184e36f2408a55281cd658ecbd3ca05cce6d6510a176eca393e26d1"
 dependencies = [
  "bytes",
- "http",
+ "http 0.2.8",
+ "pin-project-lite",
+]
+
+[[package]]
+name = "http-body"
+version = "1.0.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "1cac85db508abc24a2e48553ba12a996e87244a0395ce011e62b37158745d643"
+dependencies = [
+ "bytes",
+ "http 1.1.0",
+]
```

Comment:
Do we need two different versions of http-body bringin two different versions of http?

Reply:
I dont think so! I will take some time to try to deduplicate these added packages.

Reply:
After spending the last several hours attempting to migrate the Unit API client from hyper 0.14 to hyper 1.3.1 I have stumbled on this: https://github.com/OpenAPITools/openapi-generator/issues/17876

Regrettably for NOW we do need both versions. The good news is that the last version of v0 is fairly recent and I get the impression it is still being supported inline with v1. We should make it a priority to migrate to the newest version of Hyper when the openapi generator supports it. In the meantime we will have some extra build dependencies.

Reply:
Heres a ticket to track that:
https://github.com/nginx/unit/issues/1245